### DISCUSSION TITLE
What's the difference between "frequency" and "repeat after" when setting up a plant reminder?

### DISCUSSION THREAD

**Discussion by igorsantos07** [MDeLuise/plant-it#283] [Q&A]
They both mean the same to me, but "repeat after" does not show in the summary on the plant details screen... :thinking:

**Comment by MDeLuise** [comment]
Hi @igorsantos07, thanks for bringing this up.

Here's a quick clarification with an example for the watering action:
- **Frequency of 3 days** means: "Notify me after 3 days if the plant hasn't been watered."
- **Repeat after 3 days** means: "If I've already received a notification about watering the plant and haven't done it yet, remind me again after 3 days."

In essence, "Repeat after" is about how soon a reminder should be sent again after the first notification, while "Frequency" is about when the first notification should be sent.

Hope this clears things up!
